configs/example.yaml

```diff
@@ -63,11 +64,11 @@ chemical_symbols:
   - C
 
 # logging
-wandb: true                                                                       # we recommend using wandb for logging, we'll turn it off here as it's optional
+wandb: true                                                                        # we recommend using wandb for logging
 wandb_project: toluene-example                                                     # project name used in wandb
 
-verbose: info                                                                      # the same as python logging, e.g. warning, info, debug, error. case insensitive
-log_batch_freq: 1000000                                                                  # batch frequency, how often to print training errors withinin the same epoch
+verbose: info                                                                      # the same as python logging, e.g. warning, info, debug, error; case insensitive
+log_batch_freq: 10                                                                 # batch frequency, how often to print training errors withinin the same epoch
 log_epoch_freq: 1                                                                  # epoch frequency, how often to print and save the model
```

Comment:
@nw13slx This currenlty says "how often to print and save the model" but I think that should only determine how often to print, and *not* the saving, right?

Reply:
yes.